ads/google/adsense.js

```diff
@@ -40,19 +41,12 @@ export function adsense(global, data) {
   global.document.body.appendChild(s);
 
   const i = global.document.createElement('ins');
-  i.setAttribute('data-ad-client', data['adClient']);
-  if (data['adSlot']) {
-    i.setAttribute('data-ad-slot', data['adSlot']);
-  }
-  if (data['adHost']) {
-    i.setAttribute('data-ad-host', data['adHost']);
-  }
-  if (data['adtest'] != null) {
-    i.setAttribute('data-adtest', data['adtest']);
-  }
-  if (data['tagOrigin']) {
-    i.setAttribute('data-tag-origin', data['tagOrigin']);
-  }
+  ['adChannel', 'adClient', 'adSlot', 'adHost', 'adTest', 'tagOrigin']
```

Comment:
I think `adTest` won't work, because `data-ad-test` doesn't exist (as far I know).

I think it should be `adtest`, because the attribute name is `data-adtest`.